renku/models/_tabulate.py

```diff
@@ -32,8 +32,16 @@ def format_cell(cell, datetime_fmt=None):
 
 def tabulate(collection, headers, datetime_fmt='%Y-%m-%d %H:%M:%S', **kwargs):
     """Pretty-print a collection."""
+    if isinstance(headers, dict):
+        attrs = headers.keys()
+        # if mapping is not specified keep original
+        names = [
+            key if value is None else value for key, value in headers.items()
```

Comment:
Do empty headers make sense? If not we can simplify to:

```suggestion
            value or key for key, value in headers.items()
```

Reply:
Imho, It makes a lot, due to partial mapping. I would actually rethink this entire solution which you initially proposed. But for now, I think it suffices.